Hardware/Hardware/CardReader/StripeCardReader/StripeCardReaderService.swift

```diff
@@ -8,6 +8,7 @@ public final class StripeCardReaderService: NSObject {
 
     private var discoveryCancellable: StripeTerminal.Cancelable?
     private var paymentCancellable: StripeTerminal.Cancelable?
+    private var refundCancellable: StripeTerminal.Cancelable?
```

Comment:
Maybe this is overthinking it, but do we need a separate Cancellable for refunds? On the one hand, a payment and a refund should be mutually exclusive, so we could get a way with a simple `cancellableOperation`, on the other hand, having two separate properties makes the code more declarative. 🤷🏼

Reply:
I like the more declarative way here... I agree we could merge them but I don't think we get any big benefit from doing that.